src/linter/cache_test.go

```diff
@@ -135,7 +135,7 @@ main();
 		// 2. Check cache "strings" hash.
 		//
 		// It catches new fields in cached types, field renames and encoding of additional named attributes.
-		wantStrings := "62321eae8ac6f753221d367a0a0e2a3190202ef49c608ceef6e03dd6b09906c64bf20a1470abef9e26b917836a68415a5459491208fb692d8773524cdbe6d239"
+		wantStrings := "dc486a1cdb1bd6dcac2742113c0eeeed28f67644704c0fda1c7229e6620d3c838c39f2cf0ce83736eea9ef23cf8818197ff7ea851f0987fdddefaa6ae26f4ef1"
```

Comment:
Updating the test but not the cache version... ![you are not paid to think](https://images-na.ssl-images-amazon.com/images/I/71atYl%2BCKoL._AC_SL1500_.jpg "Mindless worker is a happy worker!")